project/profiles/capacity4more/modules/c4m/content/c4m_content_group/plugins/content_types/c4m_content_group.inc

```diff
@@ -48,3 +48,15 @@ function c4m_content_group_details() {
 
   return drupal_render($node_view);
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function c4m_content_group_preprocess_node(&$variables) {
+  if ($variables['view_mode'] != 'group_dashboard') {
+    return;
+  }
+
+  $variables['theme_hook_suggestions'] = ['node__' . $variables['node']->type . '__group_dashboard'];
```

Comment:
@nirgn975 
Do we have to override this array?
why not just adding the suggestion ?